config/opendatahub-io/org.yaml

```diff
@@ -315,6 +316,7 @@ orgs:
         description: UX team working on the ODH Dashboard component
         maintainers:
         - andrewballantyne
+        - dgutride
```

Comment:
Not sure this is 100% necessary... but a backup for me to help out Kyle wouldn't hurt.